xml/installation-installation-gui_installer.xml

```diff
@@ -159,21 +159,20 @@
     </para>
     <substeps>
      <step>
-      <para>
-       Update the zypper configuration file.
-      </para>
-  <screen>
-<?dbsuse-fo font-size="0.65em"?>
-sudo sed -i -e "s,.*solver.allowVendorChange.*,solver.allowVendorChange = true," /etc/zypp/zypp.conf
-  </screen>
-     </step>
-     <step>
-      <para>
-       Mount the repo as <literal>DC8S</literal>:
+      <para vendor="suse">
+       Mount the repository as <literal>&cloud;</literal>:
```

Comment:
You will not make friends with your users if you advise that they mount anything into a dir with a super-long name with spaces in the middle. I think "DC8S", despite being cryptic is a much better name, though, how about just "cloud-install-repo" or so? -- a bit long, but easy to easy to memorize and we won't ever need to update that name because there is no version number in there.

Then, you can also remove the vendor profiling from this para.